src/identity-providers/IdentityProvidersSection.tsx

```diff
@@ -72,7 +72,7 @@ export const IdentityProvidersSection = () => {
   const DetailLink = (identityProvider: IdentityProviderRepresentation) => (
     <Link
       key={identityProvider.providerId}
-      to={`/${realm}/identity-providers/${identityProvider.providerId}/settings`}
+      to={`/${realm}/identity-providers/${identityProvider.providerId}/${identityProvider.alias}/settings`}
     >
       {identityProvider.alias}
```

Comment:
If `displayName` is set, this should render the `displayName` here like in the old console.
```suggestion
      {identityProvider.displayName
        ? identityProvider.displayName
        : identityProvider.alias}
```

Reply:
@jenny-s51 this is true, but i was going to fix that in a future PR. i wanted to get this in ASAP because i was not able to edit existing id providers, so it makes it difficult to test that all the advanced fields are working. so i'd like to handle that outside of this PR if that's ok.

Reply:
Added this, thank you!